modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -419,20 +419,8 @@ public static class DistributionZones {
         /** Distribution zones group. */
         public static final ErrorGroup DISTRIBUTION_ZONES_ERR_GROUP = registerGroup("DISTRZONES", (short) 10);
```

Comment:
I understand that this should not apply to your changes, it would be great to register these groups as independently of modules as possible.